### PR TITLE
add Alex as EasyBuild maintainer

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: docs/Maintainers.rst [hunk 1]
```diff
@@ -68,6 +68,10 @@ The EasyBuild maintainers all have admin access to the different EasyBuild GitHu
 
   * GitHub: `@migueldiascosta <https://github.com/migueldiascosta>`_ - e-mail: ``migueldiascosta (at) nus.edu.sg`` - IRC/Slack: ``migueldiascosta``
 
+* **Alex Domingo** (`Free University of Brussels <https://hpc.vub.be>`_)
```
Comment (lexming) [inline docs/Maintainers.rst]:
```suggestion
* **Alex Domingo** (`Vrije Universiteit Brussel <https://hpc.vub.be>`_)
```

Comment (lexming) [reply docs/Maintainers.rst]: We are told to not use the english translation to avoid confusion with ULB

Comment (boegel) [reply docs/Maintainers.rst]: OK, then please fix for @wpoely86 too...

Comment (boegel) [reply docs/Maintainers.rst]: Actually, I'll do that myself...

**Comment by lexming** [comment]
Going in, thanks @boegel!

**Comment by lexming** [comment]
Going in, thanks @boegel!

**Pull request merged**